src/modules/services/components/FilePicker/constraints.js

```
@@ -72,7 +74,7 @@ export const getActionDisabledState = (actionConfig, selectedItems) => {
       const maxFileSize = actionConfig.maxFileSize
       if (
         typeof maxFileSize === 'number' &&
-        Number(selectedItem.size) > maxFileSize
+        selectedItem.size > maxFileSize
       ) {
```

ESLint

[error] Replace `⏎········typeof·maxFileSize·===·'number'·&&⏎········selectedItem.size·>·maxFileSize⏎······` with `typeof·maxFileSize·===·'number'·&&·selectedItem.size·>·maxFileSize`

(prettier/prettier)